akka-remote/src/main/scala/akka/remote/artery/Codecs.scala

```diff
@@ -84,7 +98,7 @@ private[remote] class Encoder(
       override protected def logSource = classOf[Encoder]
 
       override def onPush(): Unit = {
-        val outboundEnvelope = grab(in)
+        val outboundEnvelope = grab(in) // may contain `context` metadata attachments
```

Comment:
It's about the metadata contained in the passed around envelope, but we'll remove that completely and @bantonsson can weave it in. so the comment will become invallid in any case